codalab/worker/worker.py

```diff
@@ -759,6 +775,19 @@ def bundle_service_reply(self, socket_id, err, message, data):
         else:
             self.bundle_service.reply(self.id, socket_id, message)
 
+    def start_stage_stats(self, uuid, stage):
+        self.runs[uuid].bundle_profile_stats[stage]['start'] = time.time()
+
+    def end_stage(self, uuid, stage):
```

Comment:
For consistency, let's rename end_stage to end_stage_stats and initstats to init_stage_stats().

Also, please add docstrings to start_stage_stats, end_stage_stats, and initstats.

Reply:
@adiprerepa can you add type annotations and docstrings to all the functions you added? See this for an example: https://github.com/codalab/codalab-worksheets/blob/a715ecbadf93bbe682dab45a3015d5195d331e1a/codalab/worker/tar_subdir_stream.py#L82

Reply:
didnt know python had those, will do

Reply:
@epicfaace are the docstrings I have right now fine or do they need to be changed?